### PR TITLE
Tickets booking

### PR DESCRIPTION
**Pull request by kc211**

### CODE REVIEW

File: src/router/index.js [hunk 1]
```diff
@@ -28,6 +30,16 @@ const router = createRouter({
...
       name:'register',
       component:RegisterView
+    },
+    {
+      path:'/shows/:id',
+      name:'shows',
+      component:Shows
+    },
+    {
+      path:'/shows/seats/:id',
```
Comment (sai-eka) [inline src/router/index.js:40]: we discussed this route differently right?

File: src/views/Shows.vue [hunk 2]
```diff
@@ -0,0 +1,194 @@
+<script setup>
+import { ref, onMounted } from "vue";
+let Movies = ref([]);
+let theatres = ref([]);
+let dates = ref([]);
+let currentDate = ref(null);
+let currentDay = ref(null);
+let currentMonth = ref(null);
```
Comment (sai-eka) [inline src/views/Shows.vue]: use undefined

File: src/components/carousels.vue
Comment (sai-eka) [inline src/components/carousels.vue:1]: follow camel case  for component names

File: src/views/Shows.vue [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+    .catch((e) => console.log(e.message));
+  fetch("http://localhost:3000/theatres")
+    .then((res) => res.json())
+    .then((data) => (theatres.value = data))
+    .catch((e) => console.log(e.message));
+
+  currentDate.value = new Date().getDate();
+  currentDay.value = Day_names[new Date().getDay()];
+  currentMonth.value = Month_names[new Date().getMonth()];
+  console.log(currentDate);
```
Comment (sai-eka) [inline src/views/Shows.vue]: remove logs

File: src/components/carousels.vue
Comment (kc211) [reply src/components/carousels.vue:1]: completed

File: src/views/Shows.vue (hunk 3 above)
Comment (kc211) [reply src/views/Shows.vue]: removed.

**Pull request merged**